multi_asset/muti_token_receiver.mligo

```diff
@@ -5,5 +5,15 @@
 
 #include "multi_token_interface.mligo"
 
+
+(*
+  This receiver implementation is WIP and may result in locked tokens.
+  The implementation needs to include entry points to either initiate tokens
+  transfer from this contract and/or add operators who can transfer tokens on
+  behalf of this contract.
+  Adding additional non `multi_token_receiver` entry points are pending on
+  LIGO support for multi entry points
```

Comment:
OK, it is pending on LIGO.

I suggest the spec should describe _how_ the safety check should be implemented. It should be implemented by `CONTRACT %on_multi_tokens_received ...`.

LIGO will at this moment be unable to implement the corrected spec, indeed. You can fix the resulting Michelson by merely adding the %on_multi_tokens_received annotation to the `CONTRACT` call, but that is awkward.

Reply:
`muti_token_receiver.mligo`
`multi_token_impl.mligo`
`simple_admin.mligo`
`multi_asset.mligo`

are not part of the specification. They are one of the possible reference implementations.
And the receiver is just a testing stub at the moment.

Reply:
> I suggest the spec should describe how the safety check should be implemented. It should be implemented by `CONTRACT %on_multi_tokens_received ...`.

Done